### ISSUE TITLE
Momentum for BatchNorm2d

### ISSUE THREAD

**Issue by ryanxingql** [cszn/KAIR#132]
I notice that the momentum for BatchNorm2d is 0.9.
https://github.com/cszn/KAIR/blob/7e51c16c6f55ff94b59c218c2af8e6b49fe0668b/models/basicblock.py#L69

However, the default momentum of BatchNorm2d is 0.1.
https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html

Should we use momentum=0.9 to construct DnCNN?

**Comment by cszn** [comment]
Both are OK for Gaussian denoising.
